src/config.rs

```diff
@@ -49,6 +51,11 @@ impl Default for Config {
                 rpc_api_key: None,
                 linkdrop_account_id: Some("testnet".parse().unwrap()),
                 faucet_url: Some("https://helper.nearprotocol.com/account".parse().unwrap()),
+                meta_transaction_relayer_url: Some(
+                    "https://relayer-sweat-testnet-7tk2cmmtcq-uc.a.run.app/relay"
+                        .parse()
+                        .unwrap(),
```

Comment:
This is not the right default. It must be None.

Reply:
Done